SPECS/golang/golang.spec

```diff
@@ -1,4 +1,6 @@
-%global bootstrap_compiler_version 1.19.12
+%global bootstrap_compiler_version_0 1.17.13
+%global bootstrap_compiler_version_1 1.21.6
+%global bootstrap_compiler_version_2 1.22.4
```

Comment:
Why go back to 17 from 19? Was 19 -> 22 not good? Or 19 -> 21 -> 22?

Reply:
Go 1.22 now requires the final point release of Go 1.20 or later for bootstrap.
And Go 1.20 requires the Go 1.17. That's why I am building with 1.4 -> 1.17 -> 1.21 -> 1.22

I hope this answers your question.

Reference
https://github.com/golang/go/issues/44505#issuecomment-783611413
https://tip.golang.org/doc/go1.22

Reply:
Thanks for the explanation. I've added a few more comments about building the final version. The main thing I'm trying to understand is why do we now need to build the final version twice?